aos_cd_jobs/common.py

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env python
 from os import getenv
-from os.path import join
+from os.path import exists, join
 
 from git import Repo
 
 JOBS_DIRECTORY = 'jobs'
 
 def initialize_repo():
     repo_dir = join(getenv('WORKSPACE'), 'aos-cd-jobs')
+    if exists(repo_dir):
+        return Repo(repo_dir)
```

Comment:
The first command in the job clears the existing workspace, so we already get a
fresh clone at every run.  This change is to reuse the repository between the
execution of `pruner.py` and `updater.py`.  The intersection between the two is
very small, so there's not much risk of one interfering with the other.